CTP/shell/src/com/navercorp/cubridqa/shell/main/Context.java

```diff
@@ -139,33 +138,32 @@ public void reload() throws IOException{
 		
 		this.toolHome = com.navercorp.cubridqa.common.CommonUtils.getEnvInFile (Constants.ENV_CTP_HOME_KEY);
 		
-		this.cleanTestCase = com.navercorp.cubridqa.common.CommonUtils.convertBoolean(getProperty("main.testcase.clean", "false"));
+		this.cleanTestCase = com.navercorp.cubridqa.common.CommonUtils.convertBoolean(getProperty(ConfigParameterConstants.TESTCASE_UPDATE_YES_OR_NO, "false"));
 		 
 		this.isWindows = getTestPlatform().equalsIgnoreCase("windows");
 		
-		this.testCaseSkipKey = getProperty("main.testcase.skip_key", "").trim().toUpperCase();
+		this.testCaseSkipKey = getProperty(ConfigParameterConstants.TESTCASE_EXCLUDE_BY_MACRO, "").trim().toUpperCase();
 		if(this.testCaseSkipKey.equals("")) {
 			this.testCaseSkipKey = null;
 		}
                  		
-		this.isSaveCorefile = getProperty("main.testing.savecorefile", "FALSE").trim().toUpperCase().equals("TRUE");
-		this.bigSpaceDir = getProperty("main.testing.bigspace_dir", "").trim();
-		this.maxRetryCount = Integer.parseInt(getProperty("max.retry.count", "0").trim());
-        this.defaultDbCharset = getProperty("main.testing.default_charset", "en_US").trim();
+		this.bigSpaceDir = getProperty(ConfigParameterConstants.LARGE_SPACE_DIR, "").trim();
+		this.maxRetryCount = Integer.parseInt(getProperty(ConfigParameterConstants.TESTCASE_RETRY_NUM, "0").trim());
+        this.defaultDbCharset = getProperty(ConfigParameterConstants.CUBRID_DB_CHARSET, "en_US").trim();
        
-		this.enableCheckDiskSpace = com.navercorp.cubridqa.common.CommonUtils.convertBoolean(getProperty("main.testing.enable_check_disk_space", "FALSE").trim());
-		this.mailNoticeTo = getProperty("main.owner.mail", "").trim();
+		this.enableCheckDiskSpace = com.navercorp.cubridqa.common.CommonUtils.convertBoolean(getProperty(ConfigParameterConstants.ENABLE_CHECK_DISK_SPACE_YES_OR_NO, "FALSE").trim());
+		this.mailNoticeTo = getProperty(ConfigParameterConstants.TEST_OWNER_EMAIL, "").trim();
 		
-		this.enableSaveNormalErrorLog = getProperty("main.testing.do_save_normal_error_log", "FALSE").trim().toUpperCase().equals("TRUE");        
+		this.enableSaveNormalErrorLog = getProperty(ConfigParameterConstants.TEST_FAILURE_BACKUP_YES_OR_NO, "FALSE").trim().toUpperCase().equals("TRUE");        
```

Comment:
It's better to use convertBoolean to convert ConfigParameterConstants.TEST_FAILURE_BACKUP_YES_OR_NO